torch/csrc/jit/runtime/static/impl.h

```diff
@@ -114,7 +114,7 @@ class TORCH_API StaticModule {
  private:
   explicit StaticModule(
       std::pair<std::shared_ptr<torch::jit::Graph>, std::shared_ptr<Module>>
-          graph_and_module,
+      graph_and_module,
```

Comment:
Could you clang-format the files?

Reply:
clang-format actually added this indent. i'll remove by hand